packages/devextreme/js/ui/context_menu/ui.context_menu.js

```diff
@@ -673,6 +677,18 @@ class ContextMenu extends MenuBase {
         return availableHeight - SUBMENU_PADDING;
     }
 
+    _dimensionChanged() {
+        $(`.${DX_MENU_ITEM_EXPANDED_CLASS}`).each((_, item) => {
+            const $item = $(item);
+            const $submenu = $item.children(`.${DX_SUBMENU_CLASS}`);
+
+            if($submenu.length) {
+                this._setSubMenuHeight($submenu, $item, true);
+                this._scrollToElement($item);
+            }
```

Comment:
[missed scenario]
Not it can look like this: 
![image](https://github.com/DevExpress/DevExtreme/assets/37047972/3e74e7b0-35f5-414e-9199-11bd65890d74)